utils/transactions.go

```diff
@@ -228,17 +267,55 @@ func FeeEstToResBoundsMap(
 	feeEstimation rpc.FeeEstimation,
 	multiplier float64,
 ) *rpc.ResourceBoundsMapping {
+	bounds := CustomFeeEstToResBoundsMap(feeEstimation, multiplier, starknetLimits)
+
+	return &bounds
+}
```

Comment:
This is from an old code but it should return by value. If this is a breaking change, please leave a `Todo` to come check for it later